### ISSUE TITLE
How to get the same results of your paper with only one GPU

### ISSUE THREAD

**Issue by zhouweii234** [hkchengrex/STCN#74]
I have only one GPU now, if I want to get the same result of your paper, what parameters should I adjust? Could you please give me some guidance?

**Comment by hkchengrex** [comment]
And also: https://stackoverflow.com/questions/62067400/understanding-accumulated-gradients-in-pytorch

**Comment by zhouweii234** [comment]
Does something else like `learning rate` or `iterations` do I need to change?

**Comment by hkchengrex** [comment]
No.
